buildpipeline/Core-Setup-Signing-Windows-x64.json

```diff
@@ -50,7 +50,7 @@
       "inputs": {
         "scriptType": "filePath",
         "scriptName": "build_projects/dotnet-host-build/build.ps1",
-        "arguments": "-Configuration $(BuildConfiguration) -Targets Prepare,CompileCoreHost",
```

Comment:
I am not enabling Portable Builds in CI for now - just for official builds. I would like to do that once we decide to pull the plug on distro specific builds.